doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -24,11 +41,44 @@ By default `runCommand` runs in a stdenv with no compiler environment, whereas [
     You have to create a file or directory `$out` for Nix to be able to run the builder successfully.
     :::
 
+[allowSubstitutes]: https://nixos.org/nix/manual/#adv-attr-allowSubstitutes
+[preferLocalBuild]: https://nixos.org/nix/manual/#adv-attr-preferLocalBuild
+
+::: {.example #ex-runcommandwith}
+# Invocation of `runCommandWith`
+
+```nix
+runCommandWith {
+  name = "example";
+  derivationArgs.nativeBuildInputs = [ cowsay ];
+} ''
+  cowsay > $out <<EOMOO
+  'runCommandWith' is a bit cumbersome,
+  so we have more-ergonomic wrappers.
+  EOMOO
+''
+```
+
+:::
+
+
+## `runCommand` and `runCommandCC` {#trivial-builder-runCommand}
+
+`runCommand :: String -> AttrSet -> String -> Derivation`
+`runCommandCC :: String -> AttrSet -> String -> Derivation`
+
+`runCommand name derivationArgs buildCommand` is a derivation which runs the specified `buildCommand`
```

Comment:
`runCommand` and `runCommandCC` are functions. We have some conventions in `lib` how to document functions.
-> https://github.com/NixOS/nixpkgs/blob/master/lib/asserts.nix
This is a little inconsistent.

Reply:
Don't look at other `pkgs.*` function documentation in the manual. We are currently refactoring a lot of things.

You should follow this guide: (starting at: "When documenting functions ...")
https://github.com/NixOS/nixpkgs/blob/667f3a73f7ed070b570c03d63275acc301e7de54/doc/README.md#documentation-conventions

All functions that you encountered and don't follow those conventions yet should be/ will be refactored.

Reply:
```suggestion
The function `runCommand` returns a derivation which runs the specified `buildCommand` when beeing built.
```